RELEASE_NOTES.md

```diff
@@ -6,6 +6,10 @@ MSA Release Notes
 * Upgrade SAML libs version so that the MSA can support the ECDSA and RSASSA-PSS signing algorithms to fulfil the eIDAS cryptographic requirements
 as laid out in the "eIDAS - Cryptographic requirements for the Interoperability Framework". 
 * Upgrade to Dropwizard 1.3.5 plus updated various other libs, specifically: ida_utils, ida_test_utils, hub_saml, saml_libs_version.
+* Enhance the MatchingResponseGenerator so that it creates a Response with a request id containing
```

Comment:
May I suggest:
Extend the existing incoming health-check capability from Verify Hub to return the following MSA configurations in the response:
  * eIDAS is enabled
  * SHA1 signature is disabled